scripts/generate-schema-types.js

```diff
@@ -16,9 +16,14 @@ async function main() {
     path.join(__dirname, '../packages/core/src/domain/telemetry/telemetryEvent.types.ts'),
     'telemetry-events-schema.json'
   )
+  await generateTypesFromSchema(
+    path.join(__dirname, '../packages/rum/src/types/session-replay.ts'),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about following file name pattern?
```suggestion
    path.join(__dirname, '../packages/rum/src/types/sessionReplay.types.ts'),
```

Reply:
I agree. I think I didn't give it too much thought, can't recall where I got the need for kebab case from.